delay_out_center_task/model.py

```diff
@@ -240,8 +249,10 @@
 
 >>> environment.exists('target')
 True
->>> model.on_enter_trial_teardown()
+>>> delta = time_timeout(model.on_enter_trial_teardown)
 Trigger: end_trial
```

Comment:
I have some concern about the absence of a `Trigger: timeout` message... but the doctests pass.